app/views/users/registrations/new.html.erb

```diff
@@ -50,7 +50,9 @@
             <div class="input-group">
               <% if Flipper.enabled?(:recaptcha) %>
                 <%= flash[:recaptcha_error] %>
-                <%= recaptcha_tags %>
+                <div style="padding-bottom: 10px;">
```

Comment:
No inline styles please. Rather add top margin to the login button. Use class "users-login-primary-button" defined in app/assets/stylesheets/users.
Also, please fix this on signup page as well